src/interfaces/node.h

```diff
@@ -266,6 +267,16 @@ class Node
 //! Return implementation of Node interface.
 std::unique_ptr<Node> MakeNode();
 
+//! Block tip (could be a header or not, depends on the subscribed signal).
+struct BlockTip
+{
+    int block_height;
+    int64_t block_time;
+    uint256 block_hash;
+};
+
+BlockTip MakeTip(uint256 block_hash, int block_height, int64_t block_time);
```

Comment:
In commit "[WalletModel] Balance and Txrecord status change polling check based on the block hash and not the height." (997129bdb45c01390bffce991b680d58c9e7d18b)

I don't think you need this function, since you can create instances just as easily with `BlockHash{height, time, hash}`

Reply:
done